metadata.rb

```diff
@@ -8,8 +8,8 @@
 issues_url 'https://github.com/cvent/octopus-deploy-cookbook/issues'
 version '0.6.5'
 
-depends 'windows', '~> 1.38'
-depends 'windows_firewall', '~> 3.0.2'
+depends 'windows', '>= 1.38.0'
+depends 'windows_firewall', '>= 3.0.2'
```

Comment:
I think this should be `'~> 3.0'` This has a dependency on 3.0 at the moment.  If that API changes then I want to update that explicitly